lib/resource-mapper.js

```diff
@@ -48,9 +69,17 @@ class ResourceMapper {
       const files = await this._readdir(folder)
 
       // Find a file with the same name (minus the dollar extension)
-      const match = files.find(f => this._removeDollarExtension(f) === filename)
+      let match = files.find(f => this._removeDollarExtension(f) === filename ||
+        (isIndex && f.startsWith(this._indexName + '.')))
```

Comment:
Just wondering here if the first case and the `isIndex` case are mutually exclusive. If they are, we might want to write this differently.